### ISSUE TITLE
Laptop screen frozen

### ISSUE THREAD

**Issue by Heisfer** [hyprwm/Hyprland#868]
Anyhow I took time to compile Hyprland commits from 0.15.1 to up and last good commit (working) is https://github.com/hyprwm/Hyprland/commit/881f8282501845f57505015e85abbe668bfb740f
After https://github.com/hyprwm/Hyprland/commit/945b4d7139366d7e52bb49913a8667c08d0e567e (wlroots dep update) up to latest my laptop screen freezes.

Edit: It freezes when I launch Hyprland

**Comment by Heisfer** [comment]
I tested with lates wlroots branch https://gitlab.freedesktop.org/wlroots/wlroots/-/tree/0c962c98cc3bc4e1acd3adf83bd527e278b5597b and still seems the issue

**Comment by primalmotion** [comment]
I also noticed random freeze, sometimes even preventing me to change tty to kill Hyprland

**Comment by Heisfer** [comment]
> if this is a wlroots regression, bisect wlroots to find the commit that introduced the issue and report it on the wlroots gitlab.

The problem is that It's right between "Good" commit wlroots and "bad commit" wlroots, and the middle ground fails to compile

**Comment by vaxerski** [comment]
report it then and list all your findings. I assume this also happens on sway?

**Comment by Heisfer** [comment]
> report it then and list all your findings. I assume this also happens on sway?

I think I would need to install sway-git version if it ( I think it uses wlroots live master? ) But will test it later this night.

**Comment by Heisfer** [comment]
Anyways tested sway-git
```
wlroots-git 0.16.0.r5801.6d90518c-1
sway-git r6977.5e514e64-1
```
And it's working fine but Hyprland is freezing

**Comment by vaxerski** [comment]
then bisect Hyprland please

**Comment by vaxerski** [comment]
fixed in 47eac4be1cde424f12e942d58813a10869bfb169
